two_factor/gateways/twilio/gateway.py

```diff
@@ -52,7 +53,14 @@ def make_call(self, device, token):
                                  url=uri, method='GET', timeout=15)
 
     def send_sms(self, device, token):
-        body = gettext('Your authentication token is %s') % token
+        """
+        send sms using custom TWILIO_MESSAGE from settings.py
+        """
+        body = getattr(
+            settings,
+            'TWILIO_MESSAGE',
+            'Your authentication token is {}'
```

Comment:
I think the default value should still be a translated string. For most people this has been fine and they might be a little annoyed when their users start getting English text messages